### PR TITLE
Add zookeeper and thrift shade modules

### PR DESCRIPTION
**Pull request by zhoujinsong**
- Add zookeeper and thrift shade modules.
- Remove mvnw tools.

### CODE REVIEW

File: amoro-shade-guava-32/src/main/resources/META-INF/NOTICE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-amoro-shade-guava-32
```
Comment (klion26) [inline amoro-shade-guava-32/src/main/resources/META-INF/NOTICE]: Why do we change this?

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -38,6 +38,7 @@ jobs:
       - name: Build
         run: |
           set -o pipefail
-          ./mvnw clean deploy ${{ env.MVN_COMMON_OPTIONS }} \
+          
+          mvn clean deploy ${{ env.MVN_COMMON_OPTIONS }} \
```
Comment (klion26) [inline .github/workflows/ci.yaml:42]: could you please share more info abouth why do we revert the `mvnw` command

File: amoro-shade-guava-32/src/main/resources/META-INF/NOTICE (hunk 1 above)
Comment (zhoujinsong) [reply amoro-shade-guava-32/src/main/resources/META-INF/NOTICE]: This is an unexpected change, I have rollbacked it.

File: .github/workflows/ci.yaml (hunk 2 above)
Comment (zhoujinsong) [reply .github/workflows/ci.yaml:42]: Maven Wrapper ensures that the project's build and dependency management use a specific version of Maven. However, the Amoro shade project currently does not have this requirement, so we may not need to introduce it.

**Review by klion26 — APPROVED** [review]
thanks for the contribution, LGTM

**Pull request merged**